server/server/src/sync/init_programs_data.rs

```diff
@@ -178,7 +181,7 @@ pub fn init_program_data(
         id: patient_schema_id.clone(),
         r#type: "JsonForms".to_string(),
         json_schema: serde_json::from_str(PATIENT_SCHEMA).unwrap(),
-        ui_schema: json!({}),
+        ui_schema: serde_json::from_str(PATIENT_UI_SCHEMA).unwrap(),
```

Comment:
Is this what you meant, @clemens-msupply ?

Reply:
yes!